examples/h5_dtransform.c

```diff
@@ -1,6 +1,5 @@
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
  * Copyright by The HDF Group.                                               *
- * Copyright by the Board of Trustees of the University of Illinois.         *
```

Comment:
Doesn't this UofI copyright need to stay for files created before 2006, when THG spinned off?  You might want to check with Les and Steve.